### PR TITLE
Limit number of topics to 200

### PR DESCRIPTION
**Pull request by davidfurey**
Because the payload to Azure gets to large with more than 200 topics

### CODE REVIEW

**Comment by DiegoVazquezNanini** [comment]
:+1: very neat

File: common/src/main/scala/models/TopicType.scala [hunk 1]
```diff
@@ -2,17 +2,19 @@ package models
...
 object TopicTypes {
-  case object Breaking extends TopicType { override def toString: String = "breaking" }
-  case object Content extends TopicType { override def toString: String = "content" }
-  case object TagContributor extends TopicType { override def toString: String = "tag-contributor" }
-  case object TagKeyword extends TopicType { override def toString: String = "tag-keyword" }
-  case object TagSeries extends TopicType { override def toString: String = "tag-series" }
-  case object TagBlog extends TopicType { override def toString: String = "tag-blog" }
-  case object FootballTeam extends TopicType { override def toString: String = "football-team" }
-  case object FootballMatch extends TopicType { override def toString: String = "football-match" }
+  object Breaking extends TopicType("breaking", 100) // scalastyle:off magic.number
```
Comment (alexduf) [inline common/src/main/scala/models/TopicType.scala]: Don't hesitate to remove the rule in the scala style conf if that gets in your way

Comment (alexduf) [inline common/src/main/scala/models/TopicType.scala]: Or to drop it completely

Comment (davidfurey) [inline common/src/main/scala/models/TopicType.scala]: I mostly like the rule.  Except when it is annoying.  I'll leave it there for now.

**Pull request merged**